src/Microsoft.DotNet.ProjectModel/WorkspaceContext.cs

```diff
@@ -280,6 +351,20 @@ private FileModelEntry<LockFile> AddLockFileEntry(string projectDirectory, FileM
             return currentEntry;
         }
 
+        private ProjectContextBuilder InitializeProjectContextBuilder(Project project)
+        {
+            var builder = new ProjectContextBuilder()
+                .WithProjectResolver(path => GetProjectCore(path)?.Model)
```

Comment:
This can happen in scenarios when the project being resolved can't be found. Before this would cause an exception. Now, it causes the project dependency to be marked as unresolved (since ProjectContext already has code to handle a null result from the resolver).